src/main/java/com/lunark/lunark/reservations/dto/ReservationResponseDto.java

```diff
@@ -0,0 +1,20 @@
+package com.lunark.lunark.reservations.dto;
+
+import com.lunark.lunark.properties.dto.PropertyResponseDto;
+import com.lunark.lunark.properties.model.Property;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDate;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class ReservationResponseDto {
+    PropertyResponseDto property;
```

Comment:
Do we need to return the entire property when creating a reservation?
```suggestion
```

Reply:
I would leave this as when the guest browses his reservations, we would display property details as well.